app/services/react-query/constants/cart.ts

```diff
@@ -0,0 +1 @@
+export const CART_QUERY_KEY = 'cart'
```

Comment:
Well, CART_QUERY_KEY is a bit special constant in our architecture, we put it inside hook itself and then just export it from there.